packages/react-components/react-components/src/index.ts

```diff
@@ -99,6 +99,7 @@ export {
   useId,
   useIsomorphicLayoutEffect,
   useIsSSR,
+  isHTMLElement,
```

Comment:
actually is there an ask to make `isHTMLElement` public?

Reply:
[PositioningImperativeAnchorTarget.stories.tsx](https://github.com/microsoft/fluentui/pull/27178/files#diff-1226a7f2be23775822060a585273ceed7d9fb0e0528575c074469a55b7db2316) is using `instanceof HTMLElement`, which eslint was complaining.

Reply:
should we use a cast for the example then? or exclude stories from this rule. This is not really a problem for apps unless you are explicitly doing iframe/multi-window rendering. I wouldn't want to assume the most crazy usage as reference documentation

Reply:
Fair enough, I'll make the changes and unblock this. I'll leave the investigation for later as it's still internal